packages/vite-plugin-cloudflare/README.md

```diff
@@ -376,7 +376,7 @@ It accepts an optional `PluginConfig` parameter.
 
 - `inspectorPort?: number | false`
 
-  Optional inspector port to use for debugging your workers, for more details on debugging see the [devtools section](#devtools). Can be set to `false` to disable the debugging inspector altogether. Defaults to `9229`.
+  Optional inspector port to use for debugging your workers, for more details on debugging see the [devtools section](#devtools). Can be set to `false` to disable the debugging inspector altogether. If the value is `0` a random inspector port is chosen. By default the vite plugin will attempt to use the first available port it can find starting at `9229`.
```

Comment:
Don't think there's any need to mention the `0` value. Would there ever be a reason to use it with the new default behaviour?

```suggestion
  Optional inspector port to use for debugging your workers, for more details on debugging see the [devtools section](#devtools). By default the vite plugin will attempt to use the first available port it can find starting at `9229`.
```

Reply:
yeah, that's a good point 👍